app/src/debug/java/org/simple/clinic/DebugClinicApp.kt

```diff
@@ -89,6 +91,12 @@ class DebugClinicApp : ClinicApp() {
         .crashReporterModule(object : CrashReporterModule() {
           override fun crashReporter() = NoOpCrashReporter()
         })
+        .patientSummaryModule(object : PatientSummaryModule() {
+          override fun providesSummaryConfig(): Single<PatientSummaryConfig> {
+            return super.providesSummaryConfig()
+                .map { it.copy(isPatientEditFeatureEnabled = true) }
```

Comment:
Oh nice! Copying super values is much better than creating the values again that I've been doing.